### DISCUSSION TITLE
Notification on the main window when a new entry is input in the chat

### DISCUSSION THREAD

**Discussion by FredNor78** [jamulussoftware/jamulus#1740] [UI Related]
Most of the time we work with the Jamulus 3.7.0 main window in the foreground and some of us have a single screen so that it is unconmfortable to also see the chat window at the same time.

Hence, having a notification by the "Chat" button in the Jamulus main window when something new is written into the chat window would be appreciated

Thanks

**Comment by dcorson-ticino-com** [comment]
Thanks for bringing this up.
The present Jamulus UI was not made for tablets.
There is some discussion of how to best accommodate tablets using Jamulus.  
Adopting a single window interface is one of the possibilities discussed, but no decision has been made to do this at this point as for many users that reduces their flexibility, for example using multiple screens.
A chat button notification is one possibility, but only solves one of the problems.
But this discussion is on the table.

**Reply by hoffie** [reply]
I think this suggestion may be valid even for non-tablet environments. Some members from our choir miss chat messages as well, but it may also be due to them having their PDF viewer open to look at scores. For them, a simple chat button notification would not help, I guess.

**Reply by dcorson-ticino-com** [reply]
At least for me, receiving a chat puts the chat window in the foreground.  ??

**Reply by hoffie** [reply]
Hm, but maybe only when the first chat message arives?

**Reply by dcorson-ticino-com** [reply]
Maybe, not 200% sure to tell the truth.
